src/Tickets.java

```diff
@@ -0,0 +1,24 @@
+import java.util.Scanner;
+
+public class Tickets {
+    public static void main(String[] args) {
+        // Find the cost of tickets
+        Scanner myObj = new Scanner(System.in);
+        System.out.print("Please enter YES/NO:\nAre you a vip client??? ");
+        String status = myObj.next();
+        System.out.print("How old are you? ");
+        int age = myObj.nextInt();
+        if (age < 16) {
+            System.out.println("You can't enter as you are under 16!!! ");
+
+        } else if (status.equals("YES") || status.equals("Yes") || status.equals("yes")) {
```

Comment:
We haven't used any String method, so you must avoid the usage of `equals` method.
And you also didn't take into account payment for ordering. So this exercise is incomplete.